admin_manual/release_notes.rst

```diff
@@ -16,6 +17,97 @@ Release Notes
+
+New events for audit logging
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+New events have been added to be used for audit logging, among others. These include *configuration changes* by administrators and users, *file comments* (*add/edit/delete*) and *updating existing public links*. When logs are forwarded to external analyzers like Splunk, administrators can check to add the new events. The latest version of the Auditing extension (*admin_audit*) is required.
+
+New command to verify and repair file checksums
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+With ownCloud 10 file integrity checking by computing and matching checksums has been introduced to ensure that transferred files arrive at their target in the exact state as their origin. In some rare cases wrong checksums can be written to the database leading to synchronization issues with e.g. the Desktop Client. To mitigate such situations a new command ``occ files:checksums:verify`` has been introduced. Executing the command will recalculate checksums for all files of a user or within a specified path on the storage and compare them with the values in the database. Naturally the command also offers an option to repair incorrect checksum values (``-r, --repair``). Please check the available options by executing ``occ files:checksums:verify --help``. Note: Executing this command might take some time depending on the file count.
+
+New config setting to specify minimum characters for sharing autocomplete
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For security reasons the default value for minimum characters to trigger the sharing autocomplete mechanism has been set to "4" (previously it was set to "2"). This is to impede user name or mail address tapping. As it is a trade-off between security and usability for some scenarios this high security level might not be desirable. Therefore the value now is configurable via the *config.php* option ``'user.search_min_length' => 4,``. Please check which value fits your needs best.
+
+New option to granularly configure public link password enforcement
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+With ownCloud 10 the *"File Drop"* feature has been merged to public link permissions. As this kind of public link does not give recipients access to any contents but just gives them the possibility to "drop" files it might not be desirable for some scenarios to enforce password protection for such shares while other public links should require setting a password. For this password enforcement for public links has been split to apply based on permissions given (*read-only, read & write, upload only/File Drop*). Please check the administration settings *'Sharing'* section and configure as desired.
```

Comment:
Did you mean "_merged with public link permissions_" instead of "_merged to public link permissions_"?

Reply:
s/to any contents/to any content/g

Reply:
How about the following update?

> This kind of public link does not give recipients access to any content, but it gives them the possibility to "drop files". As a result, it might not always be desirable to enforce password protection for such shares. Given that, passwords for public links can now be set based on one of several permissions (*read-only*, *read & write*, *upload only/File Drop*). Please check the administration settings "*Sharing*" section and configure as desired.

Reply:
changed, thx